src/MIDebugEngine/Engine.Impl/DebuggedProcess.cs

```diff
@@ -491,7 +491,17 @@ private List<LaunchCommand> GetInitializeCommands()
             string escappedSearchPath = string.Join(pathEntrySeperator, _launchOptions.GetSOLibSearchPath().Select(path => EscapePath(path, ignoreSpaces: true)));
             if (!string.IsNullOrWhiteSpace(escappedSearchPath))
             {
-                commands.Add(new LaunchCommand("-gdb-set solib-search-path \"" + escappedSearchPath + pathEntrySeperator + "\"", ResourceStrings.SettingSymbolSearchPath));
+                if (_launchOptions.DebuggerMIMode == MIMode.Gdb)
```

Comment:
I will try it to make sure.

From: Gregg Miskelly [mailto:notifications@github.com]
Sent: Wednesday, March 23, 2016 12:46 PM
To: Microsoft/MIEngine <MIEngine@noreply.github.com>
Cc: Jackson Davis <Jackson.Davis@microsoft.com>
Subject: Re: [MIEngine] Fix for issues with paths in spaces (#236)


In src/MIDebugEngine/Engine.Impl/DebuggedProcess.cs<https://github.com/Microsoft/MIEngine/pull/236#discussion_r57224462>:

> @@ -491,7 +491,17 @@ private List<LaunchCommand> GetInitializeCommands()

>              string escappedSearchPath = string.Join(pathEntrySeperator, _launchOptions.GetSOLibSearchPath().Select(path => EscapePath(path, ignoreSpaces: true)));

>              if (!string.IsNullOrWhiteSpace(escappedSearchPath))

>              {

> -                commands.Add(new LaunchCommand("-gdb-set solib-search-path \"" + escappedSearchPath + pathEntrySeperator + "\"", ResourceStrings.SettingSymbolSearchPath));

> +                if (_launchOptions.DebuggerMIMode == MIMode.Gdb)

Do you know if we want the quotes with GDB on Windows?

—
You are receiving this because you authored the thread.
Reply to this email directly or view it on GitHub<https://github.com/Microsoft/MIEngine/pull/236/files/df45e52af635b7bf772198c52516f166408d3de9#r57224462>